index.js

```diff
@@ -31,6 +31,25 @@ $(document).ready(function() {
     });
 });
 
+// opens modal when img is clicked
+$(document).on('click', "img",function(){
+    $("#modal").css("display", "block");
```

Comment:
Divs are `display: block` by default. We can make the JS a bit cleaner by instead doing
```js
$('modal').show();
```
and
```js
$('modal').hide();
```
in the three functions!